docker/fdb_to_sqlite.py

```diff
@@ -0,0 +1,217 @@
+"""Module for converting a FDB database to a SQLite database"""
```

Comment:
Is there a reason why these python modules are being included statically as opposed to installing https://github.com/lcdr/utils via pip (and PRing any necessary additions to support the necessary operation methods used here, eg headless for pkextractor)?

Reply:
A better idea maybe is to apply the changes back to https://github.com/TheNoim/DarkflameServer and then add it as a submodule to the repo?

Reply:
What do you mean by "apply the changes back to https://github.com/TheNoim/DarkflameServer"?

Reply:
The python scripts are heavily modified to work headless. I originally started to make this changes on my fork, but after some time I just copied them in the directory and stopped pushing the changes to the utils fork. With "applying" the changes back I just meant, push the new changes to the fork.

Reply:
Right, my proposal is to just get the original utils repo to a point where it supports headless execution. (Alternatively, it might be nice to extend https://github.com/Xiphoseer/assembly_rs instead and migrate away from the python utils)

Reply:
Sadly, I still don't know rust, so somewhere else need to update assembly. But, I also think we should replace the python tool. But for a first version I think it is ok to have the python files in the repo.

Reply:
Yeah in my first pass at this I actually did it as a submodule. I think the changes you've made to that repo can be a PR @TheNoim and then we can submodule the repo here.